examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -18,7 +18,7 @@
 // scalastyle:off println
 package org.apache.spark.examples.streaming
 
-import scala.collection.mutable.LinkedList
+import scala.collection.mutable.ArrayBuffer
```

Comment:
Would a ListBuffer be a better fit here? This is only an example but is a O(1) prepend data structure better suited here?